### PR TITLE
Rfe trello 2694 selenium c status bar not removed on iphone 12 safari

### PR DESCRIPTION
**Pull request by at-ylazarenko**

### CODE REVIEW

File: Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs [hunk 1]
```diff
@@ -173,6 +172,9 @@ private static void InitDeviceRegionsTable_()
                 { new Size(2160, 1620), new Rectangle[]{ new Rectangle(0, 140, 2160, 1480) } },
 
                 { new Size(1640, 2360), new Rectangle[]{ new Rectangle(0, 149, 1640, 2211) } },
+
+                // iPhone 12
+                { new Size(1170, 2532), new[]{ new Rectangle(0, 290, 1170, 2242) } },
```
Comment (itaibh) [inline Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs]: You also need portrait mode.

File: Tests/Test.Eyes.Selenium.DotNet/TestBrowserStack.cs [hunk 2]
```diff
@@ -51,5 +52,40 @@ public void TestIE11(StitchModes stitchMode)
                 eyes.Abort();
             }
         }
+
+        [TestCase(StitchModes.CSS)]
+        [TestCase(StitchModes.Scroll)]
+        public void SafariTest(StitchModes stitchMode)
```
Comment (itaibh) [inline Tests/Test.Eyes.Selenium.DotNet/TestBrowserStack.cs]: No need to add another test. There are already stitching tests for safari. You'll just need to add the device in the data provider.

Also there's a crop unit test. You'll need to take the original output from BrowserStack and the expected output and add them as resources on the correct places.

Comment (at-ylazarenko) [reply Tests/Test.Eyes.Selenium.DotNet/TestBrowserStack.cs]: @itaibh 
I can see the some existing code which uses SafariOptions but this code is not in use. Could you please point me tests you mentioned about?
Can we go one time together and place the resources on the correct places?

File: Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs (hunk 1 above)
Comment (at-ylazarenko) [reply Eyes.Selenium.DotNet/Capture/SafariScreenshotImageProvider.cs]: Do you mean horizontal alignment?

File: Tests/Test.Eyes.Selenium.DotNet/TestBrowserStack.cs (hunk 2 above)
Comment (itaibh) [reply Tests/Test.Eyes.Selenium.DotNet/TestBrowserStack.cs]: check out https://github.com/applitools/Eyes.Sdk.DotNet/blob/a23576edef9a7e665e868e2f2fdb6ed5cd33e1ac/Tests/Test.Eyes.Selenium.DotNet/TestMobileDevices.cs#L172-L174

**Review by itaibh — APPROVED** [review]

**Pull request merged**